src/components/shared/LibraryTypeSelect.vue

```diff
@@ -1,42 +1,37 @@
 <template>
-  <div>
-    <traction-form-group
-      label="Library Type"
-      label-for="library-type"
-      :label-cols="labelCols"
-      :label-align="labelAlign"
-      label-size="sm"
-    >
-      <traction-select
-        id="library-type"
-        :class="getClass()"
-        :value="libraryType"
-        :options="libraryTypes"
-        @input="handleInput"
-      ></traction-select>
-    </traction-form-group>
-  </div>
+  <traction-field-group label="Library Type" label-for="library-type">
```

Comment:
This change means the field looks slightly out of place in:
1) PacbioSamplesExtractionReception - which will be removed in short order
2) The request metadata edit page - as the style of the label doesn't quite match up. However this is subtle, and I decided to leave the updating of that component to a later story.

Reply:
In its other contexts the updated style slots in just fine.